OnlineShopUI/online-shop-ui/src/components/Search.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import './Search.scss';
+import Login from './Login';
+
+import IconSearch from '../img/icon-search.png';
+import Card from '../img/icon-card.png';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Fade from '@mui/material/Fade';
+
+const Search = () => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
```

Comment:
Add some intendants for clean code, name the var 'anchorElement', and why we are setting null as default value? I don't understand the logic. Moreover, we can cast to Boolean using `anchorElement!!`.
We need to have a small talk about this part of code